assets/dev/js/editor/components/ai/components/prompt-suggestions.js

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography, Stack } from '@elementor/ui';
+import StyledChip from './ui/styled-chip';
+
+const PromptSuggestions = ( props ) => {
+	return (
+		<Box>
+			<Typography variant="subtitle1" color="text.secondary">
+				{ __( 'Suggested prompts:', 'elementor' ) }
```

Comment:
same as above, not sure if the colon should be in the translation

Reply:
Done.